src/main/java/oss/fosslight/service/impl/VerificationServiceImpl.java

```diff
@@ -973,18 +973,14 @@ public Map<String, Object> processVerification(Map<Object, Object> map, T2File f
 			// depth가 낮은 readme 파일을 구하기 위해 sort
 			if (packagingFileIdx == 1){ // packageFile에서 readMe File은 첫번째 file에서만 찾음.
 //				List<String> sortList = new ArrayList<>(deCompResultMap.keySet());
-				Collections.sort(readmePathList, new Comparator<String>() {
-
-					@Override
-					public int compare(String arg1, String arg2) {
-						if (arg1.split("\\/").length > arg2.split("\\/").length) {
-							return 1;
-						} else if (arg1.split("\\/").length < arg2.split("\\/").length) {
-							return -1;
-						} else {
-							return arg1.compareTo(arg2);
-						}
-					};
+				Collections.sort(readmePathList, (arg1, arg2) -> {
+					if (arg1.split("\\/").length > arg2.split("\\/").length) {
+						return 1;
+					} else if (arg1.split("\\/").length < arg2.split("\\/").length) {
+						return -1;
+					} else {
+						return arg1.compareTo(arg2);
+					}

```

Comment:
@kssumin Since each condition includes a `return`, I think the `else` statement  is unnecessary. WDYT?

```suggestion
				Collections.sort(readmePathList, (arg1, arg2) -> {
					if (arg1.split("\\/").length > arg2.split("\\/").length) {
						return 1;
					}
					if (arg1.split("\\/").length < arg2.split("\\/").length) {
						return -1;
					}
					return arg1.compareTo(arg2);

```

Reply:
Oh, Thank you!